bootstrap/handlers/secret.go

```diff
@@ -103,6 +103,7 @@ func SecureProviderBootstrapHandler(
 // If a token file is present it will override the Authentication.AuthToken value.
 func getSecretConfig(secretStoreInfo config.SecretStoreInfo, tokenLoader authtokenloader.AuthTokenLoader) (types.SecretConfig, error) {
 	secretConfig := types.SecretConfig{
+		Type:                    secretStoreInfo.Type,
```

Comment:
could you add a bit of description about what this `Type` mean?  developer might have hard time to figure out what type should be used...

Reply:
Done